metabase/metabase
# add basic static combo chart built with echarts

### Description

Adds basic static combo chart built with ECharts, the code moved from the prototype branch. Many visualization settings are not working on this PR but it still should render provided data.

### How to verify

1. Send a test dashboard subscription with any line/area/bar/combo
2. Ensure the data somehow rendered

### Checklist

- [n/a] Tests have been added/updated to cover changes in this PR
